src/views/Maintenancepage.vue

```diff
@@ -0,0 +1,81 @@
+            />
+            <div class="maintenance-page_description level is-mobile">
+                <div class="maintenance-page_title">
+                    {{ $t('views.maintenance.title') }}
+                </div>
+                <div class="maintenance-page_subtitle">
+                    {{ $t('views.maintenance.subtitle') }}
+                </div>
+            </div>
+        </div>
+    </main>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import mainStore from '@/store/main-store/MainStore';
+@Component({
+    name: 'Maintenance',
+    components: {},
+})
+export default class Maintenance extends Vue {
```

Comment:
@gonzalo-plaza There are two problems jejeje on the one hand, you are calling this component Maintenancepage.vue and all components are calling in UpperCamelCase, so this component should called MaintenancePage.vue, on the other hand, if this component is calling MaintenancePage why you declare the class Maintenance? and the name from the component Maintenance? The name from the component and the name should be same that the name from the file